app/components/user_avatars_stack/user_avatar/index.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+
+import UserAvatar from './user_avatar';
+
+import type UserModel from '@typings/database/models/servers/user';
+
+const enhanced = withObservables([], ({user}: {user: UserModel}) => {
```

Comment:
Probably cannot happen, but (I think) if you don't pass `'user'` to the list, if the same component is called with a different user object, it won't re-render. Or at least that is what I understand from the documentation on `withObservables`.

0/5 whether it needs to change or stay like this. Looking a bit further it seems like the key of the element is the user id, so it should never consider the same component if the user is different...